projects/go-lib/src/lib/components/go-button/go-button.component.scss

```diff
@@ -52,6 +52,7 @@
 
 .go-button--primary {
   background: $ui-color-primary;
+  border: 1px solid $ui-color-primary;
```

Comment:
We'll want to account for the hover/focus states on each variant. The background color changes, we'll want to change the color of the border to correspond to those changes.

Reply:
Additionally, the `disabled` state needs addressed as well.